src/components/WalletCongresSenate/RemarkForm/index.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { SelectInput, TextInput } from '../../InputComponents';
+import styles from '../../Modals/styles.module.scss';
+
+const remarkOptions = {
+  category: [
+    {
+      value: 'marketingAndPr',
+      display: 'Marketing and PR',
+      index: 0,
+    },
+    {
+      value: 'diplomacy',
+      display: 'Diplomacy',
+      index: 1,
+    },
+    {
+      value: 'it',
+      display: 'It',
+      index: 2,
+    },
+    {
+      value: 'legal',
+      display: 'Legal',
+      index: 3,
+    },
+    {
+      value: 'administration',
+      display: 'Administration',
+      index: 4,
+    },
+    {
+      value: 'settlement',
+      display: 'Settlement',
+      index: 5,
+    },
+    {
+      value: 'other',
+      display: 'Other',
+      index: 6,
+    },
+  ],
+};
+
+export default function RemarkForm({ register, indexItem, errors }) {
+  const categoryName = indexItem ? `category${indexItem}` : 'category';
+  const projectName = indexItem ? `project${indexItem}` : 'project';
+  const supplierName = indexItem ? `supplier${indexItem}` : 'supplier';
+  const descriptionName = indexItem ? `description${indexItem}` : 'description';
+  const amountInUsd = indexItem ? `amountInUsd${indexItem}` : 'amountInUsd';
+  return (
+    <>
+      <div className={styles.title}>Category</div>
+      <SelectInput
+        register={register}
+        options={remarkOptions.category}
+        name={categoryName}
+        selected={remarkOptions.category[0]}
+      />
+
+      <div className={styles.title}>Project</div>
+      <TextInput
+        register={register}
+        name={projectName}
+        errorTitle="Project"
+        placeholder="Project"
+        required
+        maxLength={{ value: 256, message: 'Max Project length is 256 chars' }}
```

Comment:
I thought max remark length in total is 256, not individual items?

Reply:
Correct, total remark length must be <= 256